### ISSUE TITLE
pretty print breaks raw output

### ISSUE THREAD

**Issue by endorama** [jzelinskie/faq#71]
Hello, I started using `faq` but I found what I think is an issue.

When using `raw` output mode, `faq` breaks trying to pretty encode the output. I would have expected that `raw` output would override all enhancements to the output, but it does not seem the case.

How to reproduce:
```
$ echo '"hello world"' | faq --input-format=json -r "."
Error: failed to encode as pretty: invalid character 'h' looking for beginning of value
```

Adding `--pretty-output=false` to the above command makes it work:

```
$ echo '"hello world"' | faq --input-format=json -r --pretty-output=false "."
hello world
```

Is this the expected `--raw-output` behaviour?

**Comment by jzelinskie** [comment]
This is definitely non-ideal behavior.

IIRC the pretty-print flag for jq is implicitly set based on other flags, so that this cannot happen (pretty can never be enabled if raw is enabled).

This seems like an easy enough fix, just add some validation to override the pretty flag when someone sets raw.

**Comment by endorama** [comment]
>
> This seems like an easy enough fix, just add some validation to override
> the pretty flag when someone sets raw
>

Happy to try to fix it, I'll give a go by next week.

>

**Comment by jzelinskie** [comment]
Oh, no pressure! Don't feel obligated just because you made the issue.

**Comment by jzelinskie** [comment]
Fixed in #73
